src/App.js

```diff
@@ -53,6 +54,32 @@ class App extends Component {
         }
     }
 
+	/**
+	 * This function will take a text input and checks if it is valid json
+	 * @param {String} jsonFile The json file entered by the user
+	 * @returns {Boolean} The determination if the file is valid json
+	 */
+	validateJSON(jsonFile) {
+		//first a quick way to check the size of the file is to check the character length
+		//Let's assume first of all that a JSON input file should be no larger than 1MB, or 1048576 characters
+		if(jsonFile.length > 1048576) {
```

Comment:
in this case, we might need to give user feedback that error is because of large json size

Reply:
That's definitely true, we should kick back an error message. (Maybe in an alert?)

Reply:
Alert would not be very nice UX-wise. Let's show it where sql errors are shown, may in different color, or even same is fine.